server/jobs/job_manager.go

```diff
@@ -70,7 +70,7 @@ func (jm *JobManager) OnConfigurationChange(env mscalendar.Env) error {
 
 	jm.registeredJobs.Range(func(k interface{}, v interface{}) bool {
 		job := v.(RegisteredJob)
-		enabled := job.isEnabledByConfig(env)
+		enabled := (job.isEnabledByConfig == nil) || job.isEnabledByConfig(env)
```

Comment:
Why not this `or` logic inside the "isEnableByConfig" logic?

Reply:
'cause providing a function for the "always on" case seemed wrong. nil is more appropriate?